Documentation/GeneralConventions/FileStructure.rst

```diff
@@ -8,219 +8,71 @@ File structure
 ==============
 
 This page explains the general file structure of a TYPO3 documentation that can
-be rendered with the :ref:`rendering toolchain <rendering-docs>`. The toolchain
-itself uses a `Sphinx theme <https://typo3-documentation.github.io/sphinx_typo3_theme/>`__
-for converting reStructuredText (reST) or Markdown to HTML.
+be rendered with the :ref:`rendering toolchain <rendering-docs>`.
 
 ..  contents:: Table of Contents:
     :backlinks: top
     :class: compact-list
     :depth: 2
     :local:
 
-
 .. _file-structure-general:
 
 General
 =======
 
 In order for the documentation to be rendered, you need at least
 
-#.  an index file in one of the following locations in weighted order:
-
-    -  :file:`Documentation/Index.rst`
-    -  :file:`Documentation/index.rst`
-    -  :file:`Documentation/Index.md`
-    -  :file:`Documentation/index.md`
-    -  :file:`README.rst`
-    -  :file:`README.md`
-
-#.  and a theme configuration file under
-
-    - :file:`Documentation/Settings.cfg`.
+*   A valid :file:`composer.json`
+*   A documentation entry point either for :ref:`Full documentation <full-documentation>`
```

Comment:
Also a README can hold full documentation (in one file). Would use the word "manual" instead of "full documentation".